examples/jsblocks/readme.md

```diff
@@ -0,0 +1,27 @@
+# jsblocks TodoMVC Example
+
+ > From simple user interfaces to complex single-page applications using faster, server-side rendered and easy to learn framework
+ >
+ > [jsblocks - jsblocks.com](http://jsblocks.com)
+
+
+## Resources
+
+- [Website](http://jsblocks.com)
+- [Documentation](http://jsblocks.com/learn)
+- [Download](http://jsblocks.com/download)
+- [GitHub Page](https://github.com/astoilkov/jsblocks)
```

Comment:
There is one interesting [DailyJS post](http://dailyjs.com/2015/05/29/the-jsblocks-framework/). I could add it.